sysboundary/antisymmetric.cpp

```diff
@@ -121,6 +125,41 @@ namespace SBC {
             mpiGrid[cells[c]]->sysBoundaryFlag = flag;
          }
       }
+
+         // Assign boundary flags to local fsgrid cells
+      const std::array<int, 3> gridDims(technicalGrid.getLocalSize());  
+      for (int k=0; k<gridDims[2]; k++) {
+         for (int j=0; j<gridDims[1]; j++) {
+            for (int i=0; i<gridDims[0]; i++) {
+               const auto& coords = technicalGrid.getPhysicalCoords(i,j,k);
+               const auto refLvl = mpiGrid.get_refinement_level(mpiGrid.get_existing_cell(coords));
```

Comment:
I believe the coords from the corner of the cell. Then when you get the existing cell, these coordinates are used for dccrgrid. This is also a corner for dccrg (at least at finest level), It would be numerically more stable to add 0.5 DXYZ to coordinates?

Reply:
Since I add the .5DXYZ on the following line, it's straightforward to do it before this call and also use the updated coords for the get_existing_cell call.

Reply:
Ah, no, it's not that straightforward since I need the refinement level to know how much is 0.5 DX

Reply:
I could add 0.5 * width of the highest refinement level cell

Reply:
see my comment, but otherwis ok